content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -291,6 +290,12 @@ please see [here](https://kubernetes.io/docs/concepts/cluster-administration/net
 ```shell
 kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/v0.10.0/Documentation/kube-flannel.yml
 ```
+Note that `flannel` works on `amd64`, `arm`, `arm64` and `ppc64le`, but until `flannel v0.11.0` is released
```

Comment:
can we open an issue to remove this line once the official release lands?

Reply:
@cmluciano 
done, https://github.com/kubernetes/kubeadm/issues/1069